### PR TITLE
fix/1850

### PR DESCRIPTION
**Pull request by hexyls**
closes https://github.com/protofire/omen-exchange/issues/1850
closes https://github.com/protofire/omen-exchange/issues/1737

### CODE REVIEW

**Comment by pimato** [comment]
I just played around with it and created two scalar markets with compound enabled and redeemed my winnings.

It looks like the message while the market is not yet resolved looks good:
Bildschirmfoto 2021-04-23 um 13 41 26

Once resolved the winning message is always showing the DAI balance at 0.00 DAI: 
Bildschirmfoto 2021-04-23 um 13 42 21

**Review by pimato — CHANGES_REQUESTED** [review]
see comment.

**Review by pimato — APPROVED** [review]
Works great now!

File: app/src/util/tools.ts [hunk 1]
```diff
@@ -358,6 +358,14 @@ export const getCTokenForToken = (token: string): string => {
   }
 }
 
+export const isCToken = (symbol: string): boolean => {
+  const tokenSymbol = symbol.toLowerCase()
+  if (tokenSymbol.startsWith('c')) {
+    return true
+  }
```
Comment (KadenZipfel) [inline app/src/util/tools.ts:365]: This could potentially cause problems... I'm thinking we could probably call compound's API instead to do this check

**Review by KadenZipfel — CHANGES_REQUESTED** [review]

File: app/src/util/tools.ts (hunk 1 above)
Comment (hexyls) [reply app/src/util/tools.ts]: Oh yeah true! This has to be sync in some places, luckily I discovered we actually have a `CompoundTokenType` so I can check against that.

**Review by jpkcambridge — APPROVED** [review]

**Review by KadenZipfel — APPROVED** [review]

**Pull request merged**